### DISCUSSION TITLE
How do i get notified about this event.?

### DISCUSSION THREAD

**Discussion by ishabhthakur** [prisma/prisma#12837] [Q&A]
#There is no timer in the website about to notified about the event

**Comment by janpio** [comment]
Which event do you refer to?

**Reply by ishabhthakur** [reply]
https://www.prisma.io/blog/mongodb-general-availability-pixnun6mffmu

this is the event Prisma + mongodb April-25to29

**Reply by janpio** [reply]
There is a website at https://www.prisma.io/mongodb-launch where you can sign up. I would expect we will use this to tell you more information when it is available. (Otherwise: Put yourself a reminder to look at that website every few days until additional information is added.)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
